src/Grpc/HttpApi/perf/Microsoft.AspNetCore.Grpc.Microbenchmarks/Microsoft.AspNetCore.Grpc.Microbenchmarks.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Microsoft.AspNetCore.Grpc.HttpApi\Microsoft.AspNetCore.Grpc.HttpApi.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Protobuf Include=".\Proto\chat.proto" GrpcServices="Server" />
+    <Protobuf Include=".\Proto\greet.proto" GrpcServices="Client" />
+
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Instead of re-adding this `@(FrameworkReference)` item, set the `$(UseAspNetCoreSharedRuntime)` property to `true`. Then ensure Build.props does _not_ build this project directly. Instead, have another project depend on Microsoft.AspNetCore.App.Ref then `<MSBuild/>` this one. InteropTests.csproj does this and seems to be working **Just Fine:tm:**. May need to get change the root `Directory.Build.*` files to import `FunctionalTestWithAssets.*` in perf projects too. Up to use whether renaming `@(TestAssetProjectReference)` is needed.

Note the `$(TestDependsOnAspNetRuntime)` property is specific to Helix testing and not applicable here.

Reply:
I removed all the `<FrameworkReference Include="Microsoft.AspNetCore.App" />` references. They aren't needed. I'm guessing it is automatically included by all projects that depend on `Grpc.AspNetCore.Server` which references it.